app/repositories/default_thresholds_repository.rb

```diff
@@ -0,0 +1,5 @@
+class DefaultThresholdsRepository
+  def stream_thresholds(stream:)
+    DefaultThreshold.find_by(sensor_name: stream.sensor_name, unit_symbol: stream.unit_symbol)
```

Comment:
I think I would move here logic returning values from stream when there is no default config available.